virtio-devices/src/block.rs

```diff
@@ -41,6 +40,7 @@ use vm_migration::{
     Migratable, MigratableError, Pausable, Snapshot, SnapshotDataSection, Snapshottable,
     Transportable,
 };
+use vm_virtio::rate_limiter::{RateLimiter, TokenType};
```

Comment:
Since the rate limiter is not virtio specific at all, and could easily be reused for non virtio devices, I think it makes sense to move it to its own crate. I'm also thinking we shouldn't add more things to `vm-virtio` as it will make the move to the upstream version harder as well.
WDYT @rbradford ?

Reply:
Definitely not in the vm-virtio crate. Leave in the virtio-devices crate for now and can move if we need it elsewhere (don't see anything on the immediate horizon though).

Reply:
No he can't leave it in `virtio-devices` because there's some cyclic dependencies otherwise.

Reply:
Okey dokey. New crate it is then. :+1:

Reply:
Got it. Will move it to its own crate.